src/load.rs

```diff
@@ -204,18 +198,26 @@ impl Loader {
                 None => break,
                 Some(s) => s,
             };
+
             match stmt {
-                Statement::Include(id) => trace::scope("include", || {
-                    self.evaluate_and_read_file(id, &[&parser.vars])
-                })?,
-                // TODO: implement scoping for subninja
-                Statement::Subninja(id) => trace::scope("subninja", || {
-                    self.evaluate_and_read_file(id, &[&parser.vars])
-                })?,
+                // TODO: Support new scope for 'include' statements.
```

Comment:
I think this should be 'subninja' here -- "include" doesn't create a new scope, while "subninja" does.